src/components/StaffGrid/index.jsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const StaffGrid = ({ staff }) => (
+  <div className="staff-grid row">
+    {staff.map(staffer => (
+      <div className="staffer-wrapper col-6 col-md-4 col-lg-3" key={staffer.uuid}>
+        <img src={staffer.profile_image_url} className="rounded-circle w-50" alt="" />
```

Comment:
I see you're putting the border-radius on the images themselves. While working on the updated program details pages I noticed that not all of the instructor images are in the preferred format (see the last instructor here https://sandbox-edx-prospectus.edx.org/micromasters/columbiax-business-analytics). So to prevent those ones looking broken I put the border-radius and width on the wrapper, and use the ```overflow-hidden``` class.

Reply:
Happy to make these changes, but on the sandbox I'm seeing the same thing as this, except with `overflow:hidden` in the css for the image wrapper

Reply:
Yeah, for the experiment on the sandbox I'm actually overriding the classes on ```img``` instead of changing universally. I should update that.

Reply:
Still not completely sure what changes you want here. When I move the `rounded-circle w-50` utility classes up to the wrapper along with `overflow-hidden`, that causes some funkiness